framework/include/interfaces/Coupleable.h

```diff
@@ -267,6 +267,33 @@ class Coupleable
   std::vector<const VariableValue *> coupledVectorTagValues(const std::string & var_name,
                                                             const std::string & tag_name) const;
 
+  /**
+   * Returns value of a coupled array variable for a given tag
+   * @param var_name Name of coupled array variable
+   * @param tag vector tag ID
+   * @param comp Component number for vector of coupled variables
```

Comment:
```suggestion
   * @param index Index of the desired variable in the vector of coupled variables
```

Reply:
similar for other routines. Would be good to update simple and vector too

Reply:
Docstrings and function arguments were updated for coupledVectorTag functions (standard, array and matrix).